### PR TITLE
Add map of Romania

### PR DESCRIPTION
**Pull request by ALEX11BR**

### CODE REVIEW

**Comment by VictorCazanave** [comment]
Thanks a lot for your contribution @ALEX11BR ! 😃 

I'll try to review it next week.

File: packages/romania/romania.svg [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+     name="Botoșani"
+     id="bt" />
+  <path
+     d="m 384.73418,355.97362 -4.49,-0.7 -0.83,-1.85 0.11,-3.02 -0.53,0.32 -1.58,-0.26 -1.58,0.16 -2.74,1.59 -0.31,-0.25 -0.36,0.19 0.11,1.45 -0.52,-0.08 -1,0.78 -0.16,2.21 2.34,1.37 0.21,1.28 -3.37,0.16 -0.25,1.1 1.02,2.05 1.29,0 0.22,0.75 0.77,0.16 0.4,-0.46 1.23,1.4 0.35,-0.27 0.67,1.24 3.03,0.88 0.15,0.59 0.39,0 0.02,-0.67 0.65,-0.05 -0.11,0.45 0.55,0.57 0.62,-0.11 0.8,1.84 1.81,-0.67 -1.56,-2.96 1.23,-2.31 0.62,0.3 1.29,-0.32 0.64,-0.86 0.43,0.32 1.4,-2.77 -4.33,-1.42 0.56,-1.27 z"
+     name="Bucharest"
+     id="b" />
```
Comment (VictorCazanave) [inline packages/romania/romania.svg:41]: Short names are OK, but I think 1 letter is too short 😄 
Could you use full names or ISO 3166-2 format for all the `id` please?

File: packages/romania/romania.svg [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+     name="Sălaj"
+     id="SJ" />
```
Comment (VictorCazanave) [inline packages/romania/romania.svg:141]: Could you fix the remaining uppercase `id` attributes please?

**Review by VictorCazanave — COMMENTED** [review]
Few small things to fix, then it's good to be merged

File: packages/romania/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# 🇷🇴️ Romania
+
+Contains the municipality of Bucharest and all the other counties of Romania:
+* Alba
```
[the remark this answers is not in the document]
Comment (ALEX11BR) [reply packages/romania/README.md:4]: The municipality of Bucharest is not a county technically, but it's at the same level, so I put it in the list description, at line 3:

> Contains the **municipality of Bucharest** and all the other counties of Romania:
> * List goes on...

**Comment by ALEX11BR** [comment]
I don't know why my commits appear under the profile sauerbraten...

**Comment by ALEX11BR** [comment]
Still new to this github pull requests thing, turns out

**Comment by mihai-ro** [comment]
@ALEX11BR,  @VictorCazanave , gentlemen, any updates on this PR? I was planning on putting up my own PR on the topic, but before doing so I decided to check pending PRs.

**Comment by VictorCazanave** [comment]
I thought it hasn't been updated, then kinda forgot about it, sorry 🙏 
I'll try to merge it in the next few days

**Comment by VictorCazanave** [comment]
> I don't know why my commits appear under the profile sauerbraten...

I guess you committed with another git user (`git config user.name` and `git config user.email`)

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @ALEX11BR for code

**Pull request merged**